azure-pipelines.yml

```diff
@@ -129,15 +114,6 @@ jobs:
       AnalyzeSymPath: $(Build.StagingDirectory)\symbols
     continueOnError: true
 
-  - task: securedevelopmentteam.vss-secure-development-tools.build-task-apiscan.APIScan@2
```

Comment:
Should we run APIScan against v3.1 binaries instead of moving it?

Reply:
Put it back